.github/workflows/fmt.yml

```diff
@@ -0,0 +1,20 @@
+name: Fmt
+
+on: [push, pull_request]
+
+jobs:
+  fmt:
+    name: Fmt
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      - name: Install rustfmt
+        run: rustup component add rustfmt
+
+      - name: Run cargo fmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
```

Comment:
Wouldn't it be better if it commited after running `cargo fmt`?

Reply:
Trick is there is ok with formatting in master branch, so it will be no fmt errors after merge and we can see it because `Fmt / Fmt (pull_request)`  passed